packages/flutter/test/material/tabs_test.dart

```diff
@@ -1758,6 +1758,97 @@ void main() {
     semantics.dispose();
   });
 
+  testWidgets('can be notified of TabBar onTap behavior', (WidgetTester tester) async {
+    int tabIndex = -1;
+    bool hasBeenTapped = false;
```

Comment:
This appears to be redundant with tabIndex != -1 and existing `expect`s already check that.